src/targets/chrome/create-chrome-target.js

```diff
@@ -237,17 +243,18 @@ function createChromeTarget(
     const tab = await launchNewTab(tabOptions);
     try {
       await withTimeout(options.chromeLoadTimeout)(tab.loadUrl(url));
+      const screenshot = await tab.captureScreenshot(selector);
+      await fs.outputFile(outputPath, screenshot);
+      return screenshot;
```

Comment:
I'm not certain about how JS will treat this, but will the `finally` block be executed even after a `return` statement?

Reply:
I think finally should always get executed. Still, I'll move the return statement for clarity.